### PR TITLE
Add building and container recipe

### PR DESCRIPTION
**Pull request by Congpy**
```
docker build -t exercise .
```
```
docker run -it exercise /bin/bash
```
** I actually have a problem. When I was running docker for the second time my SSD drive on C:\ run out of space because of an update. And no matter what I do, I cannot clear it anymore. (I am not allowed to move the appdata folder to another drive which covers the most space.) So I tried to install docker on D:\ but so far it was futile. This is why I cannot run docker anymore, I am kind of in the dark here, about knowing whether my code works or not.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+RUN chmod +x /usr/local/bin/inittimezone
+CMD ./usr/local/bin/inittimezone
+
+RUN apt-get update -y && apt-get install -y build-essential cmake git wget vim libboost-all-dev libdeal.ii-dev
+
+RUN mkdir exercise && cd exercise
+RUN git clone https://github.com/jbeder/yaml-cpp ./yaml-cpp
+RUN mkdir ./yaml-cpp/build && cd ./yaml-cpp/build && cmake .. && make && make install
+RUN cd .. && git clone https://github.com/Congpy/cmake-exercise-wt2223.git
+RUN mkdir ./build && cd ./build && cmake ../ && make
```
Comment (uekerman) [inline Dockerfile:13]:
```suggestion
RUN git clone https://github.com/Congpy/cmake-exercise-wt2223.git
RUN cd cmake-exercise-wt2223 && mkdir ./build && cd ./build && cmake ../ && make
```
Every `RUN` command starts from root again.

**Review by uekerman — APPROVED** [review]
Needed a small tweak to get it to run.
Otherwise, looks good :+1:

Fingers crossed that you manage to fix your issue :/

**Comment by uekerman** [comment]
I am closing the PR to have a better overview of what is still open.

File: Dockerfile (hunk 1 above)
Comment (Congpy) [reply Dockerfile:13]: Yes I have formatted Windows 11 and installed it in HDD rather than SSD, thank you ! :)

**Pull request closed without merging**